### ISSUE TITLE
When the game ends?

### ISSUE THREAD

**Issue by SanaHOUCH** [enferas/Mgh-game#2]
The game is over when a player loses either by being reduced to two pieces OR by being unable to move.

**Comment by anbanasmou** [comment]
we need to add the two rules .(player can remove from a mill if he don t have a choice .and the flying thing )

**Comment by SanaHOUCH** [comment]
Yeah, A player reaches the flying phase when he is left with only 3 pieces. in that case he can move any piece to any empty point on the board.

**Comment by Oussamab21** [comment]
We need also to make draw rules as a perfection no? I use this link for rules please every one look at it and we need to agree if we will use it as a rule reference or no (if we think that some rules are different to implement )
http://www.yourturnmyturn.com/rules/mill.php
